README.md

```diff
@@ -1,6 +1,306 @@
+They are the Broker and its colocated CNS Object store.
+It should be noted that the Broker can be implemented to run anywhere
+They fill the role of our micro-service provider
+It is only for the purpose of this demo that we decided deploy the Broker and the CNS Cluster in the same location
+
+
+A second system will be the locally running Kubernetes cluster on which with Service-Catalog is deployed
+This cluster will be our micro-server client
+Please refer to the [command flow diagram](docs/diagram/control-diag.md) for a more in depth look at these systems.
+
+## Setup
+### Step 0: Preparing environment
+- Clone [Kubernetes](https://github.com/kubernetes)
+- Clone [Service-Catalog](https://github.com/kubernetes-incubator/service-catalog)
+- Install [Google SDK](https://cloud.google.com/sdk/) and add `gcloud` to `PATH`
+You must have an active Google Cloud Platform account.
+- Configure `gcloud` user, zone, and project
+These are detected by the deployment script and required for setup.
+
+
+`# gcloud config set account <user account>`
+
+`# gcloud config set project <project>`
+
+`# gcloud config set compute/zone <zone>`
+
+
+### Step 1: Deploy Gluster-Kubernetes cluster in Google Compute Engine (GCE)
+This step sets up the **External Service Provider** portion of our topology (see [diagram](docs/diagram/control-diag.md)).
+
+To kick off deployment, run `gk-cluster-deploy/deploy/cluster/gk-up.sh`
+The script has a number of configurable variables relative to GCE Instance settings
+They can be found in `deploy/cluster/lib/config.sh`  These can be overridden inline with `gk-up.sh` or as environment variables. To run the script with out pausing run`gk-up.sh -y`.
+
+Runtime takes around 5 to 10 minutes
+Go get some coffee.
+
+Alternatively, you can manually deploy the GCE portion by following [these instuctions](//TODO) // TODO
+
+When deployment completes, commands to ssh into the master node and the URL of the CNS Object Broker will be output. **Note the URL and PORT.**
+
+### Step 2: Deploy the Service-Catalog
+This step sets up the **K8s Cluster** portion of our topology (see [diagram](docs/diagram/control-diag.md)).
+
+*In a separate terminal:*
+1. Change directories to the `service-catalog` repository.
+
+2. Follow the [Service-Catalog Installation instructions](https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/introduction.md#installation)
+Once the Service-Catalog is deployed, return here.
+
+
+## Using the Service Catalog
+
+**STOP!** If you have made it this far, it is assumed that you now have
+
+- A 4 node Kube cluster in GCE, running:
+  - Gluster-Kubernetes and its S3 components.
+  - Our CNS Object Broker.
+
+You can check the status of all these components at once by executing
+
+`# gcloud compute ssh <master node name> --command="kubectl get pod,svc --all-namespaces"`
+
+### Create the *ServiceBroker* (API Object)
+
+Change to the `cns-object-broker` directory created when cloning the repo.
```

Comment:
"SSH to the GCE master node and change to the `cns-object-broker` directory ..."

Reply:
This is all done locally.  The example yamls are inside `cns-object-broker/exampes/`.